tss/node/cmd/tssnode/cmd.go

```diff
@@ -12,7 +12,6 @@ func Command() *cobra.Command {
 	}
 
 	cmd.AddCommand(StartCmd())
-	cmd.AddCommand(InitConfigTemplate())
 
 	cmd.PersistentFlags().StringP("config", "c", "config", "configuration file with extension")
```

Comment:
we dont need it. It is declared in parent command

Reply:
solved